provisioning/roles/cuttlefish-app/tasks/main.yml

```diff
@@ -92,3 +92,11 @@
     owner: root
     group: root
     mode: 0744
+
+- name: Create old email archiving cronjob
+  copy:
```

Comment:
We should use the Ansible [`cron` module](https://docs.ansible.com/ansible/cron_module.html) for this. Here's an example of a [rake task being set up using this module](https://github.com/openaustralia/morph/blob/3a5e7c5693e0811a59246dc869b9692d04524b56/provisioning/roles/morph-app/tasks/main.yml#L132-L133) for morph.